File: curso-python/ex066.py
soma = 0
cont = 0
while True: 
    n = int(input("Digite um valor (999 para parar): "))
    if n == 999:
        print("saindo...")
        break
    soma += n
    cont += 1

print(f"a soma dos {cont} valores foi {soma}")
    

File: javascript-exercicios/ex066.js
const prompt = require("prompt-sync")();

let soma = 0
let cont = 0

while (true){
    let n = Number(prompt("digite um valor (999 para parar): "))
    if (n == 999){
        console.log("saindo...")
        break
    }
    soma += n
    cont += 1
}
console.log(`a soma dos ${cont} valores foi ${soma}`)